dynamiclinks/app/src/main/java/com/google/firebase/quickstart/deeplinks/kotlin/MainActivity.kt

```diff
@@ -106,6 +118,29 @@ class MainActivity : AppCompatActivity() {
         return link.uri
     }
 
+    @VisibleForTesting
+    fun buildShortLinkFromParams(deepLink: Uri, minVersion: Int) {
+        val uriPrefix = getString(R.string.dynamic_links_uri_prefix)
+
+        // Set dynamic link parameters:
+        //  * URI prefix (required)
+        //  * Android Parameters (required)
+        //  * Deep link
+        Firebase.dynamicLinks.shortLinkAsync {
+        link = deepLink
+        domainUriPrefix = uriPrefix
+        androidParameters {
+            minimumVersion = minVersion
+        }
+        }.addOnSuccessListener { (shortLink, flowchartLink) ->
+        val shortLinkTextView = findViewById<TextView>(R.id.shortLinkViewSend);
+        shortLinkTextView.text = shortLink.toString();
+        }.addOnFailureListener(this) { e ->
+        Log.e(TAG, e.toString());
+        throw java.lang.Error(e.toString());
+        }
```

Comment:
The indentation is looking odd here, I wonder how `ktlint` didn't catch that (maybe we didn't configure it to warn about indentation improvements).
We also don't need to throw the exception:
```suggestion
        Firebase.dynamicLinks.shortLinkAsync {
            link = deepLink
            domainUriPrefix = uriPrefix
            androidParameters {
                minimumVersion = minVersion
            }
        }.addOnSuccessListener { (shortLink, flowchartLink) ->
            val shortLinkTextView = findViewById<TextView>(R.id.shortLinkViewSend);
            shortLinkTextView.text = shortLink.toString();
        }.addOnFailureListener(this) { e ->
            Log.e(TAG, e.toString());
        }
```

Reply:
Yea that is odd. I wonder if we could add an editorConfig to remove this issues with indentation. Any thoughts?

Reply:
<kbd>cmd</kbd> + <kbd>opt</kbd> + <kbd>L</kbd> or <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd> usually does the trick for me on Android Studio.
But if you know the editConfig to enforce it, feel free to send a PR and I'll review it :)